src/components/DataView/DataViewTHead.tsx

```diff
@@ -15,13 +15,16 @@ import { DataViewProps } from "./DataViewTypes";
 
 const StyledWrapper = styled.thead`
 	text-align: left;
+	&>tr {

```

Comment:
Same comment as on the TBody, applying the padding to the tr doesn't actually apply the padding as you can see here: 
![image](https://user-images.githubusercontent.com/23409284/203443797-f568485e-946f-4665-89e6-fe64599466e3.png)

It has to go inside the first and last TD's of the row.